### PR TITLE
feat: add sbom-tools package

### PR DESCRIPTION
**Pull request by mcasimir**

### CODE REVIEW

File: packages/sbom-tools/src/load-bundle-info.ts [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+export async function loadBundleInfo<
+  T extends { name: string; version: string }
+>(files: string[]): Promise<T[]> {
+  const data = (
+    await Promise.all(
+      files.map(async (fileName) =>
+        JSON.parse(await fs.readFile(fileName, 'utf-8'))
+      )
+    )
+  ).reduce((acc: any[], curr: any) => [...acc, ...curr], []);
```
Comment (gribnoysup) [inline packages/sbom-tools/src/load-bundle-info.ts:13]:
```suggestion
  ).flat();
```

File: packages/sbom-tools/src/commands/generate-vulnerability-report.ts [hunk 2]
```diff
@@ -0,0 +1,187 @@
+import { promises as fs } from 'fs';
+
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const snykPolicy = require('snyk-policy');
```
[the remark this answers is not in the document]
Comment (mcasimir) [reply packages/sbom-tools/src/commands/generate-vulnerability-report.ts:4]: doesn't have types or type defs

[the remark this answers is not in the document]
Comment (mcasimir) [reply packages/sbom-tools/src/commands/generate-vulnerability-report.ts:4]: though .. is written in typescript 😕

[the remark this answers is not in the document]
Comment (gribnoysup) [reply packages/sbom-tools/src/commands/generate-vulnerability-report.ts:4]: Gotcha! Unfortunate, but makes sense

File: packages/sbom-tools/src/get-package-info.ts [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+  // Normalize the 'contributors' list.
+  packageJSON.contributors = [
+    ...new Set([packageJSON.author, packageJSON.contributors].flat()),
+  ].filter(Boolean) as any;
+
+  const licenseFiles = await Promise.all(
+    (
+      await fs.readdir(packagePath)
+    )
+      .filter((filename) => licenseRegexp.exec(filename))
```
Comment (addaleax) [inline packages/sbom-tools/src/get-package-info.ts:88]:
```suggestion
      .filter((filename) => licenseRegexp.test(filename))
```

**Review by addaleax — APPROVED** [review]

**Pull request merged**